packages/app/src/containers/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './LandingPage';
+export * from './SignupPage';
+export * from './RegistrationDetails';
```

Comment:
i get a build failure on this line is there a file called `/RegistrationDetails`
`Unable to resolve "./RegistrationDetails" from "src/containers/index.ts"
Failed building JavaScript bundle.`

Reply:
@petermsouzajr This issue is resolved